config/krakend/settings/frinx_frontend_settings.json

```diff
@@ -2,15 +2,11 @@
     "endpoints": [
         {
             "endpoint": "/",
-            "url_pattern": "/frinxui"
+            "url_pattern": "/"
```

Comment:
just defines what ui to show on / might nor be frinxui at all

Reply:
this means that "/" will be mapped to "/" in our frinx-frontend but if someone wants different UI to be shown, he needs to put this part into different JSON (specific for that UI)